drivers/i2c/i2c_nrfx_twi.c

```diff
@@ -237,43 +237,49 @@ static int twi_nrfx_pm_control(struct device *dev, u32_t ctrl_command,
 }
 #endif /* CONFIG_DEVICE_POWER_MANAGEMENT */
 
+/*
+ * Factors preventing use of DT_INST:
+ *
+ * - per-instance configuration options like CONFIG_I2C_0_NRF_TWI.
```

Comment:
I actually don't see how these options prevent the use of DT_INST here.

Reply:
Because `idx` is an SoC number, not a DT_INST instance number

Reply:
My point is that it is not because of these options that it is impossible to use DT_INST macros here. This code could not use these options at all, we could even remove them from Kconfig, but this would not make it possible to use DT_INST macros here. Therefore, to me this comment is a bit misleading.

Reply:
I'm keeping my silly opinions to myself now ;). Thanks for review.